### PR TITLE
Problem: no test for new gov proposal format

### PR DESCRIPTION
**Pull request by macong-cdc**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: integration_tests/cosmoscli.py [hunk 1]
```diff
@@ -1129,6 +1129,21 @@ def gov_propose_token_mapping_change(self, proposal, **kwargs):
             )
         )
 
+    def submit_gov_proposal(self, proposal, **kwargs):
```
Comment (mmsqe) [inline integration_tests/cosmoscli.py:1132]: Maybe we could rename [existing](https://github.com/crypto-org-chain/cronos/blob/f3e063a9cd41db9451879ea802acf5a45db1f8de/integration_tests/cosmoscli.py#L1117) one as `submit_gov_proposal`

File: integration_tests/test_basic.py [hunk 2]
```diff
@@ -723,3 +727,57 @@ def test_replay_protection(cronos):
...
+    rsp = cli.submit_gov_proposal(proposal_file, from_="community")
+    assert rsp["code"] == 0, rsp["raw_log"]
+
+    # get proposal_id
+    ev = parse_events(rsp["logs"])["submit_proposal"]
+    print(ev)
+    proposal_id = ev["proposal_id"]
+    print("gov proposal submitted", proposal_id)
+
+    # not sure why, but sometimes can't find the proposal immediatelly
```
Comment (yihuang) [inline integration_tests/test_basic.py:765]: yeah, that's normal, because tx broadcast only wait for included in mempool.

**Review by yihuang — APPROVED** [review]
LGTM

File: integration_tests/test_basic.py (hunk 2 above)
Comment (macong-cdc) [reply integration_tests/test_basic.py]: This comment is removed in the latest commit.

File: integration_tests/cosmoscli.py (hunk 1 above)
Comment (macong-cdc) [reply integration_tests/cosmoscli.py:1120]: Yup, `gov_propose_token_mapping_change` has been removed with other refactoring accordingly.

**Comment by yihuang** [comment]
@macong-cdc can you sign the commits.

**Comment by macong-cdc** [comment]
> @macong-cdc can you sign the commits.

Just signed the commits and pushed. Sorry, I forgot `git rebase` removes the commits' signature.

**Pull request merged**